src/test/java/com/wikia/webdriver/elements/mercury/pages/login/SignInPage.java

```diff
@@ -0,0 +1,15 @@
+package com.wikia.webdriver.elements.mercury.pages.login;
+
+import com.wikia.webdriver.elements.mercury.components.login.JoinButtons;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.elements.mercury.components.login.JoinButtons'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)